Benchmarks doesn't build

```
$ cmake .. -DBENCHMARKING=ON && make               
-- -DCMAKE_BUILD_TYPE=Debug
-- -DTESTING=ON
-- -DBENCHMARKING=ON
-- -DDOCS=OFF

 *** EVERYTHING FINE HERE ***

[ 97%] Building CXX object benchmark/CMakeFiles/cache_benchmark.dir/ametsuchi/cache_benchmark.cc.o
/home/kitsu/dev/Work/iroha-ametsuchi/benchmark/ametsuchi/cache_benchmark.cc:18:29: fatal error: ametsuchi/cache.h: No such file or directory
 #include <ametsuchi/cache.h>
                             ^
compilation terminated.
make[2]: *** [benchmark/CMakeFiles/cache_benchmark.dir/build.make:63: benchmark/CMakeFiles/cache_benchmark.dir/ametsuchi/cache_benchmark.cc.o] Error 1
make[1]: *** [CMakeFiles/Makefile2:461: benchmark/CMakeFiles/cache_benchmark.dir/all] Error 2
make: *** [Makefile:95: all] Error 2
```

--- a/benchmark/ametsuchi/cache_benchmark.cc
+++ b/benchmark/ametsuchi/cache_benchmark.cc
@@ -15,7 +15,7 @@
  * limitations under the License.
  */
 
-#include <ametsuchi/cache.h>
+#include <ametsuchi/pager/cache.h>
 #include <benchmark/benchmark.h>
 #include <stdint.h>
 #include <algorithm>
@@ -47,7 +47,7 @@ static void Cache_RandomAccess(benchmark::State& state) {
   auto sequence = generateAccessSequence(0, state.range(0), state.range(1));
   auto begin = sequence.begin();
   auto end = sequence.end();
-  ametsuchi::Cache<int, Page> cache(state.range(2));
+  ametsuchi::pager::Cache<int, Page> cache(state.range(2));
   uint64_t items = 0;
   while (state.KeepRunning()) {
     // simulate cache

